python/hail/table.py

```diff
@@ -1403,7 +1396,7 @@ def collect(self):
         :obj:`list` of :class:`.Struct`
             List of rows.
         """
-        return hl.tarray(self.row.dtype)._from_json(self._jt.collectJSON())
+        return hl.tarray(self._row.dtype)._from_json(self._jt.collectJSON())
```

Comment:
why these changes?

Reply:
we should use public methods when we can. row === _row in this case, right?

Reply:
Yeah. This was left over from when row was actually row_value and I forgot to change back.